integration-tests/tests/package.json

```diff
@@ -20,35 +20,35 @@
 		"build": {
 			"command": "tsc",
 			"dependencies": [
-				"../../packages/realm:bundle",
+				"../../packages/realm:build:ts",
 				"build-dependencies"
 			]
 		},
 		"start": {
 			"command": "mocha --watch",
 			"dependencies": [
-				"../../packages/realm:bundle",
+				"../../packages/realm:build:ts",
 				"build-dependencies"
 			]
 		},
 		"test": {
 			"command": "mocha --exit",
 			"dependencies": [
-				"../../packages/realm:bundle",
+				"../../packages/realm:build:ts",
 				"build-dependencies"
 			]
 		},
 		"coverage": {
 			"command": "nyc mocha --exit",
 			"dependencies": [
-				"../../packages/realm:bundle:coverage",
+				"../../packages/realm:build:ts:coverage",
```

Comment:
I think this is missing from `realm/package.json`

Reply:
True - that was half baked 🙃